styles.css

```diff
@@ -0,0 +1,116 @@
+body {
+  width: 1020px;
+  margin: 0 auto;
+  font-size: 1.2em;
+  font-family: Cursive;
+  background-color: ghostwhite;
+}
+
+body a {
+  text-decoration: none;
+  color: #000080;
+}
+
+h1 {
+  font-size: 3em;
+  font-weight: bold;
+}
+
+h2 {
+  margin-bottom: 10px;
+  padding: 15px 10px 0px;
+  font-size: 2em;
+  font-weight: bold;
+  border-top: 1px solid gray;
+}
+
+table {
+  font-size: 14px;
+  font-family: "Lucida Sans Unicode";
+  text-align: justify;
+  border-spacing: 0;
+}
+
+td {
+  padding: 10px 20px;
+  background: #F8E391;
+  border-color: white;
+  border-style: solid;
+  border-width: 1px;
+}
+
+.contacts-grid {
+  display: grid;
+  margin: 30px;
+  grid-template-columns: repeat(2, 15px) 150px;
+  grid-template-rows: repeat(3, 30px);
+  align-items: center;
+  justify-items: start;
+  font-size: 1.2em;
+  font-family: Lato;
+  font-weight: 700;
+  color: #bebebe;
+}
+
+.contacts-grid img:hover {
+  -webkit-transform: scale(1.5);
+  -ms-transform: scale(1.5);
+  transform: scale(1.5);
+}
+
+.contacts-grid>img {
```

Comment:
Before and after the combinator between selectors (for example, p > a) one space is put. See more in this guide: http://codeguide.academy/html-css.html#html-language